Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.py

```diff
@@ -0,0 +1,212 @@
+    incidents = []
+    items = incident_data['products']
+
+    for item in items:
+        item['updated_at'] = item['updated_at'][:-5]
+        incident_time = dateparser.parse(item['updated_at'])
+        incident = {
+            'name': item['title'],
+            'occurred': incident_time.strftime('%Y-%m-%dT%H:%M:%SZ'),
+            'rawJSON': json.dumps(item)
+        }
+
+        incidents.append(incident)
+
+        if incident_time > max_time:
+            max_time = incident_time
+
+    next_run = {'time': max_time.strftime('%Y-%m-%dT%H:%M:%S')}
+    incidents.reverse()
+
+    return incidents, next_run
+
+
+def main() -> None:
+    """Main method used to run actions."""
+    try:
+        demisto_params = demisto.params()
+        base_url = demisto_params.get("url", "").rstrip("/")
+        verify_ssl = not demisto_params.get("insecure", False)
+        proxy = demisto_params.get("proxy", False)
+        headers = {
+            "accept": "*/*",
+            "API-TOKEN": demisto_params["apitoken"],
+            "API-SECRET": demisto_params["apikey"],
+        }
+        client = Client(
+            base_url=base_url, verify=verify_ssl, headers=headers, proxy=proxy
+        )
+        commands: Dict[str, Callable] = {
+            'dragos-get-indicators': get_indicators,
+            'dragos-get-full-report': get_report,
+            'dragos-get-ioc-csv': get_csv,
+            'dragos-get-stix2': get_stix
+        }
+        command = demisto.command()
+        if command == "test-module":
+            try:
+                client.whoami()
+                return_results("ok")
+            except Exception as err:
+                message = str(err)
+                try:
+                    error = json.loads(str(err).split("\n")[1])
+                    if "fail" in error.get("result", {}).get("status", ""):
+                        message = error.get("result", {})["message"]
+                except Exception:
+                    message = (
+                        "Unknown error. Please verify that the API"
+                        f" URL, Token and Key are correctly configured. RAW Error: {err}"
+                    )
+                raise DemistoException(f"Failed due to - {message}")
+        elif command == 'fetch-incidents':
+            first_fetch = demisto_params.get('first_fetch', '24 hours').strip()
+            incidents, next_run = fetch_incidents(
+                client=client,
+                last_run=demisto.getLastRun(),
+                first_fetch=first_fetch
+            )
+            demisto.setLastRun(next_run)
+            demisto.incidents(incidents)
+        elif command in commands:
+            return_results(commands[command](client, demisto.args()))
+
+    except Exception as e:
+        return_error(
+            f"Failed to execute {demisto.command()} command. "
+            f"Error: {str(e)}"
+        )
+
+
+if __name__ in ("__main__", "__builtin__", "builtins"):
+    main()
```

Comment:
please fix the [linters errors](https://app.circleci.com/pipelines/github/demisto/content/165012/workflows/474a6fb0-6246-4d96-a986-064ca2bb53bc/jobs/520935/parallel-runs/0/steps/0-107)